website/content/en/docs/main/tutorials/iperf/index.md

```diff
@@ -88,168 +78,32 @@ Install iPerf3 (client and server) on the clusters:
-
-   *Client cluster*:
-
-   ```sh
-   kubectl rollout status deployment cl-controlplane -n clusterlink-system
-   kubectl rollout status deployment cl-dataplane -n clusterlink-system
-   ```
-
-   *Server cluster*:
-
-   ```sh
-   kubectl rollout status deployment cl-controlplane -n clusterlink-system
-   kubectl rollout status deployment cl-dataplane -n clusterlink-system
-   ```
-
-   {{% expand summary="Sample output" %}}
-
-   ```sh
-   deployment "cl-controlplane" successfully rolled out
-   deployment "cl-dataplane" successfully rolled out
-   ```
-
-   {{% /expand %}}
+{{% include "../../../../static/files/tutorials/deploy-clusterlink.md" %}}
 
 ## Enable cross-cluster access
 
 In this step, we enable connectivity access between the iPerf3 client and server.
  For each step, you have an example demonstrating how to apply the command from a
  file or providing the complete custom resource (CR) associated with the command.
 
-Note that the provided YAML configuration files refer to environment variables
- (defined below) that should be set when running the tutorial. The values are
- replaced in the YAMLs using `envsubst` utility.
-
-{{% expand summary="Installing `envsubst` on macOS" %}}
-In case `envsubst` does not exist, you can install it with:
-
-```sh
-brew install gettext
-brew link --force gettext
-```
-
-{{% /expand %}}
+{{% include "../../../../static/files/tutorials/envsubst.md" %}}
 
 ### Set-up peers
 
-Add the remote peer to each cluster:
-
-*Client cluster*:
-
-{{< tabpane text=true >}}
-{{% tab header="File" %}}
-
-```sh
-export SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' server-control-plane`
-curl -s $IPERF3_FILES/clusterlink/peer-server.yaml | envsubst | kubectl apply -f -
-```
-
-{{% /tab %}}
-{{% tab header="Full CR" %}}
-
-```sh
-export SERVER_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' server-control-plane`
-echo "
-apiVersion: clusterlink.net/v1alpha1
-kind: Peer
-metadata:
-  name: server
-  namespace: clusterlink-system
-spec:
-  gateways:
-    - host: "${SERVER_IP}"
-      port: 30443
-" | kubectl apply -f -
-```
-
-{{% /tab %}}
-{{< /tabpane >}}
-
-*Server cluster*:
-
-{{< tabpane text=true >}}
-{{% tab header="File" %}}
-
-```sh
-export CLIENT_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' client-control-plane`
-curl -s $IPERF3_FILES/clusterlink/peer-client.yaml | envsubst | kubectl apply -f -
-```
-
-{{% /tab %}}
-{{% tab header="Full CR" %}}
-
-```sh
-export CLIENT_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' client-control-plane`
-echo "
-apiVersion: clusterlink.net/v1alpha1
-kind: Peer
-metadata:
-  name: client
-  namespace: clusterlink-system
-spec:
-  gateways:
-    - host: "${CLIENT_IP}"
-      port: 30443
-" | kubectl apply -f -
-```
-
-{{% /tab %}}
-{{< /tabpane >}}
+{{% include "../../../../static/files/tutorials/peer.md" %}}
```

Comment:
please remove the use of `include` shortcode for docsy's `readfile`, if possible.
Also, please use absolute path as in other places.